### DISCUSSION TITLE
🔗 Allow to set a delay for automatically closing the (mega) Menu Item Content (Duplicate of #21353)

### DISCUSSION THREAD

**Discussion by yvrsalazar** [elementor/elementor#23946] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

The new Menu element does not factor in the user's hover intent and causes dropdowns to close while the user is trying to move the cursor to the drop-down menu.

Elementor's WordPress Menu element does this by default.

### Describe the solution you'd like

Like with the animation duration, add a 'Close Delay'  option for the user to enter a custom duration.

![image](https://github.com/elementor/elementor/assets/50907075/1d417be4-2833-4cd6-80c7-f2be15fa439f)

**Comment by nicholaszein** [comment]
Hello there, @yvrsalazar !

Thank you for your submission! 🙏 

### 🔗 🚀 There is already a similar request submitted by someone here #21353

Feel free to continue the conversation and cast your **⬆️ Upvote** there! 👉

### 🔍 Please, in the future, remember to check if there isn't any thread similar to yours already opened. 

Kind regards
